jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -462,19 +498,19 @@ def mos_meta_parser(app, data_obj, ids=None, spectra=False, sp1d=False):
             filters = [x[0].header.get("FILTER", FALLBACK_NAME) for x in data_obj]
             gratings = [x[0].header.get("GRATING", FALLBACK_NAME) for x in data_obj]
 
-            filters_gratings = [(f+'/'+g) for f, g in zip(filters, gratings)]
+            filters_gratings = [(f+'/'+g) for f, g in zip(filters, gratings)]*repeat
```

Comment:
Easier to read if you add whitespace around the `*`. Same comment applies to all similar occurrences.

```suggestion
            filters_gratings = [(f+'/'+g) for f, g in zip(filters, gratings)] * repeat
```